components/admin/lessons/AdminLessonNav.tsx

```diff
@@ -1,18 +1,22 @@
-import React from 'react'
+import React, { FunctionComponent } from 'react'
 import styles from '../../../scss/adminLessonNav.module.scss'
 
 type Props = {
   tabs: { tabName: string; urlPageName: string }[]
-  render: (navItem: { tabName: string; urlPageName: string }) => JSX.Element
+  Component: FunctionComponent<{ tabName: string; urlPageName: string }>
 }
 
-const AdminLessonNav: React.FC<Props> = ({ tabs, render }: Props) => {
+const AdminLessonNav: React.FC<Props> = ({ tabs, Component }: Props) => {
   return (
     <div>
       <div className={styles.lessons_tabsNav}>
         <div className={styles.lessons__tabsNav__nav}>
           {tabs.map((tab, i) => (
-            <React.Fragment key={i}>{render(tab)}</React.Fragment>
+            <Component
+              key={i}
+              tabName={tab.tabName}
```

Comment:
Adding on to the comment above, could we do `<Component key={i} tab={tab} />` This makes more sense to me because `tabName` and `urlPageName` is not used for any logic in here so we can make it generic